src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url('./fonts/Avenir-Heavy.ttf');
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, 'Helvetica', sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.search-bar {
+  display: block;
+  min-width: 100%;
```

Comment:
why min-width? default value for block element is 100% of parent block

Reply:
fix this comment please